docker/dev/Dockerfile

```diff
@@ -158,8 +165,11 @@ USER ubuntu
 # Install Parflow
 # -----------------------------------------------------------------------------
 
-COPY --chown=ubuntu:ubuntu . $BASE_PATH/parflow/src
+# Do the pip installs on the earliest layer for faster iterative builds
+COPY --chown=ubuntu:ubuntu ./docker/dev/rebuild.sh ~/rebuild.sh
```

Comment:
Additionally, I've copied the rebuild script we use within the container into a convenient location